metadata.json

```diff
@@ -13,15 +13,15 @@
   "operatingsystem_support": [
     {
       "operatingsystem": "ubuntu",
-      "operatingsystemrelease": ["14.04", "16.04", "18.04"]
```

Comment:
why are we dropping the latest version?

Reply:
I added these right before bumping to 2.0.3 as unit tests passed but `puppet-agent-*:latest` images contain puppet agent v6 which the e2e smoketest uncovered as not working once you have a full server up and an agent so we can't really "support" them. I think if you run puppet 5 manually it might work but the bottom line is that I don't know how this metadata ties into their marketplace and funnels into end users to be comfortable with adding that for this release.

Reply:
ok - so you are not removing something that's been here for a while, it's something you recently added that you no longer think is a good idea

Reply:
correct